src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -155,6 +156,8 @@ uct_rdmacm_cm_device_context_init(uct_rdmacm_cm_device_context_t *ctx,
     }
 
     ctx->use_reserved_qpn = 0;
+    ctx->num_dummy_qps    = 0;
```

Comment:
can we avoid duplication with line 144? initialize it earlier/later?

Reply:
done